app_config.py

```diff
@@ -12,4 +12,4 @@
 LOG_TO_FILE = True
 LOG_TO_CONSOLE = True
 
-MINIMUM_WAIT_TIME = 60
+MINIMUM_WAIT_TIME = 10
```

Comment:
can you add unit to varaible

Reply:
Do you mean as a comment beside the variable identifying its unit, like "# in seconds"?

Reply:
yes